lib/src/entrypoint.dart

```diff
@@ -107,6 +107,30 @@ class Entrypoint {
 
   LockFile? _lockFile;
 
+  late PackageConfig packageConfig = () {
```

Comment:
Maybe, give a documentation comment that warns that this throws `DataException` if loading the package config fails.
And maybe, indicate that this should be used when this exception is handled, or desirable, for example `dart run`.

Reply:
Done